osu.Game/Input/Bindings/GlobalActionContainer.cs

```diff
@@ -66,6 +68,8 @@ public enum GlobalAction
         DecreaseVolume,
         [Description("Toggle mute")]
         ToggleMute,
+        [Description("Take screenshot")]
+        TakeScreenshot,

```

Comment:
there's no way you tested this. this will offset all actions below where you added it.